README.md

```diff
@@ -1 +1,52 @@
-# Module1Capstone
\ No newline at end of file
+# Capstone Project
+
+> In this capstone project you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://emyrue.github.io/Module1Capstone/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps: 
+1. Open your terminal and navigate to the folder you would like to put these files.
+2. Type these commands into the terminal: 
+ - git clone https://github.com/emyrue/Module1Capstone.git
+ - cd Portfolio
+ - open index.html
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@emyrue](https://github.com/emyrue)
+- Twitter: [@EmyrueRobertson](https://twitter.com/EmyrueRobertson)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emily-robertson-70a2bb22a/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please fix the broken MIT link for a completely professional README file and repo 😄